cupy/core/core.pyx

```diff
@@ -735,6 +735,17 @@ cdef class ndarray:
         """
         return _statistics._ndarray_argmax(self, axis, out, dtype, keepdims)
 
```

Comment:
Could you add a newline for PEP8?

Reply:
Flake8 throws error as too many blank lines (2) after line 736. Is PEP8 different for this?

Reply:
https://github.com/cupy/cupy/pull/2222/files#r301018458

Reply:
Got it. Will fix that one